timestamp.md

```diff
@@ -14,19 +14,30 @@ In CockroachDB, `TIMESTAMP WITHOUT TIME ZONE` is an alias for `TIMESTAMP` and `T
 
 ## Formats
 
-When inserting into a `TIMESTAMP` column, use one of the following formats:
+Constant values of type `TIMESTAMP`/`TIMESTAMPTZ` must be expressed using an
+[interpreted literal](sql-constants.html#interpreted-literals) or a
+string literal
+[annotated with](sql-expressions.html#explicitly-typed-expressions)
+type `TIMESTAMP`/`TIMESTAMPTZ` or
+[coerced to](sql-expressions.html#explicit-type-coercions) type
+`TIMESTAMP`/`TIMESTAMPTZ`.
+
+`TIMESTAMP` constants can be expressed using the
```

Comment:
DATE/INTERVAL/TIMESTAMP constants are not lexical elements and thus not distinguishable as separate constant categories in the page "SQL constants". However I agree that a mutual xref was missing; I added it.